src/media/css/detail/more-info.styl

```diff
@@ -0,0 +1,40 @@
+/*
+    The info section below the reviews section, featuring detailed app
+    information (category, package size) and content ratings.
+*/
+@import '../lib';
+
+.app-more-metadata {
+    > div {
```

Comment:
I can add a class, but it'd be adding some KBs to the templates the way it's set up. I can comment though